apps/web/src/components/comments.tsx

```diff
@@ -12,11 +21,44 @@ import { zodForm } from 'src/utils/validation';
 
 import { AttachmentEditorList } from './attachments-editor';
 import { Button } from './button';
+import { Card } from './card';
 import { MemberAvatarName } from './member-avatar-name';
 import { Message } from './message';
 import { createRichEditor, RichEditorToolbar } from './rich-editor';
 
-const T = createTranslate('common');
+const T = createTranslate('components.comments');
+
+export function Comments(props: { type: CommentEntityType; entityId: string }) {
+  const t = T.useTranslate();
+  const invalidate = useInvalidateApi();
+
+  const query = useQuery(() =>
+    apiQuery('getComments', { query: { type: props.type, entityId: props.entityId } }),
```

Comment:
I'd like to do something like this, but I wonder if we'd lose reactivity.

```suggestion
    apiQuery('getComments', { query: pick(props, ['type', 'entityId']) }),
```